iphone/Maps/UI/Settings/MWMSettingsViewController.mm

```diff
@@ -29,7 +29,7 @@ @interface MWMSettingsViewController ()<SettingsTableViewSwitchCellDelegate>
 @property(weak, nonatomic) IBOutlet SettingsTableViewSwitchCell *autoZoomCell;
 @property(weak, nonatomic) IBOutlet SettingsTableViewLinkCell *voiceInstructionsCell;
 @property(weak, nonatomic) IBOutlet SettingsTableViewLinkCell *drivingOptionsCell;
-
+@property(copy, nonatomic) NSString *headerIdentifier;
```

Comment:
Where is it used?

Reply:
My mistake :( . Will fix that
Thanks for being attentive!